iarrc/src/ol_reliable/ol_reliable.cpp

```diff
@@ -83,9 +83,9 @@ int main(int argc, char **argv) {
     int num_regions;
     pnh.param("num_regions", num_regions, 3);
 
-    int regionWidth = 640 / num_regions;
+    int regionWidth = 160 / num_regions;
```

Comment:
This planner is brittle to image size. Not sure if we need to fix this for this PR though

Reply:
Shouldn't need to worry about this for this PR. Hopefully we won't be using this planner ever again.